package.json

```diff
@@ -6,6 +6,10 @@
   "license": "MIT",
   "main": "index.js",
   "private": true,
+  "engines": {
+    "node": "10.7.0",
+    "yarn": "1.9.2"
```

Comment:
Shouldn't we use `"node": ">=10.7.0"` and `"yarn": ">=1.9.2"`?
I wasn't able to install and build this branch because I was using node 10.8.0.

Reply:
those fields are supposed to be advisory-only, so if they are preventing the install, something is wrong and it is better to just remove them

Reply:
Yeah, here's a screenshot of what's happening here: 

![captura de tela de 2018-08-06 11-04-53](https://user-images.githubusercontent.com/17071199/43721239-a954b02e-9968-11e8-94a7-01a01bcc856a.png)

Reply:
ok, removed